openverse_api/catalog/settings.py

```diff
@@ -53,7 +60,7 @@
 }
 SHORT_URL_PATH_WHITELIST = ["/v1/list", "/v1/images/"]
 
-USE_S3 = os.getenv("USE_S3", False)
+USE_S3 = os.getenv("USE_S3", "False") in TRUE_STRINGS
```

Comment:
```suggestion
USE_S3 = os.getenv("USE_S3", "False").lower() in TRUE_STRINGS
```

Should there be a small utility function to interpret boolean env values? In some sense the `https://django-environ.readthedocs.io/en/latest/` might be helpful here anyway as it already does this, but something like `bool_env("USE_S3" False)` might make sense to have as this code is repeated several times.

Reply:
Reminds me of this nightmare code on the frontend: 
https://github.com/WordPress/openverse-frontend/blob/main/src/utils/string-to-boolean.js

Reply:
Used `decouple` in e58d890.